### ISSUE TITLE
Tooltips break hover events for underlying controls

### ISSUE THREAD

**Issue by Tio-Henry** [godotengine/godot#73926]
### Godot version

4.0.rc5

### System information

Win10, AMD Radeon 6570, 15.201.1151.1008, GLES3

### Issue description

https://user-images.githubusercontent.com/54606139/221365890-7bb75953-8f31-47b5-a412-19d7c2774a4d.mp4

### Steps to reproduce

https://user-images.githubusercontent.com/54606139/221365890-7bb75953-8f31-47b5-a412-19d7c2774a4d.mp4

### Minimal reproduction project

N/A

**Comment by YuriSizov** [comment]
This may have been reported before, but indeed. Tooltips break hover events on controls (and this is not the only case of that).

**Comment by YuriSizov** [comment]
Not strictly related to the editor, but adding a label anyway to make it easier to find.

**Comment by FlankeR-46** [comment]
Good day. I won't create another issue because the situations are very similar.
Situation:
In the Project settings, we set the "WINDOWED" mode (mark 1).
We hover the mouse over any component that shows a tooltip (mark 2).
When launching the program through the code, we set the "MAXIMIZED" mode or any "FULLSCREEN" mode (mark 3).
We get a hanging tooltip on our screen, which is removed via ALT+TAB (mark 4).
Sometimes it's unpleasant.
![image](https://user-images.githubusercontent.com/126438573/222054136-9ef42040-4d1b-4d19-b403-5c224750f61b.png)

**Comment by Sauermann** [comment]
Bisecting this issue shows that it got introduced in #72104

**Comment by akien-mga** [comment]
CC @bruvzg

**Comment by golddotasksquestions** [comment]
I can confirm the issue still being present in both Godot 4.0.3 stable and 4.1 Beta3.

**Comment by duianto** [comment]
Godot v4.1.rc2 - Windows 10.0.19045 - Vulkan (Compatibility) - GeForce GTX 550 Ti () - Intel(R) Core(TM) i3-2100 CPU @ 3.10GHz (4 Threads)

On the Anchor presets popup.
Mouse clicking:
- Outside a tooltip.
  - Only closes the tooltip.
- On a tooltip
  - Closes the tooltip
  - and sends the mouse click to the button under the mouse pointer.

https://github.com/godotengine/godot/assets/13420573/be76fe4b-b2ea-4206-82a7-7adf52e12694


If a button's tooltip is visible when another button is clicked.
Then, the first click on the new button only hides the tooltip.

A second click is required, but if it's too slow and the new button's tooltip appeared.
Then, the second click hid the new button's tooltip.

A third click is required to click the button.

It's likely that the second click is slow because it was unexpected that the first click didn't click the button.

https://github.com/godotengine/godot/assets/13420573/0c246568-bbc0-4b57-a884-e8a258c621b0

### Expected

The buttons shouldn't be blocked by a tooltip.

And the tooltips should probably update as soon as the mouse is moved to a new button.

### Alternative to a floating tooltip
Instead of a floating tooltip that hides the buttons behind it.

![Xy9F3GOlnh](https://github.com/godotengine/godot/assets/13420573/86087719-99b0-437e-804d-7bf8f09bbd7a)

Maybe the anchor presets popup could have an empty text row,
that shows the current tooltip text when the mouse is over a button.

If implemented, then it also could be added to the popup:
Sizing settings for children of a Container node.

![Godot_v4 1-rc2_win64_VzrawJ5muv](https://github.com/godotengine/godot/assets/13420573/0f478732-0f9c-412a-8bdb-12ebb0623518)

The Anchor presets button `Set to Current Ratio`
and the Sizing settings for children popups check boxes.
Might still need the floating tooltips, since the text is wider than the popup widths.

![qMS5EziXm0](https://github.com/godotengine/godot/assets/13420573/21143a7d-858c-44b5-a489-35a455f3c43d)

![8avsarXvmO](https://github.com/godotengine/godot/assets/13420573/7cfc13d9-b5b3-4a50-8ef7-732055bac921)

**Comment by Tio-Henry** [comment]
The problem remains version 4.1, but a little different

https://github.com/godotengine/godot/assets/54606139/193e29c7-2c83-47d0-816b-6fbc62f3882d

**Comment by JekSun97** [comment]
The problem still remained in 4.3 dev 5

**Comment by kitbdev** [comment]
Fixed in 4.3.beta2
